e2e_samples/config_driven_data_pipelines/README.md

```diff
@@ -148,6 +149,118 @@ spark.stop()
 The sample table is like below: 
  <img src= "./docs/images/Sample table output.png" width="2000px"/>

```

Comment:
Hi @buhongw7583c Looks like the link is broken. Would you try ![architecture](./docs/images/Sample table output.png =1000x)?

Reply:
I can see this picture at my side.  How about refresh the page? This belongs to the previous PR and last time siliang also got some picture cannot displayed issue and after refresh, they are back. 


<img width="1075" alt="image" src="https://user-images.githubusercontent.com/37413937/185036235-1a630d87-2bd6-44da-8f7e-33686ca8e5a4.png">